include/uxr/agent/transport/serial/baud_rate_table_linux.h

```diff
@@ -0,0 +1,160 @@
+// Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef UXR_AGENT_TRANSPORT_SERIAL_BAUD_RATE_TABLE_H
+#define UXR_AGENT_TRANSPORT_SERIAL_BAUD_RATE_TABLE_H
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <termios.h>
+
+speed_t getBaudRate(const char* baudrate_str)
+{
+    speed_t rv;
+    if (0 == strcmp(baudrate_str, "0"))
```

Comment:
What about using a map<std::string, speed_t> initialized with all these value?
The function could scale better, and it will use less line to maintain. Similar to the table found in the test part.

Reply:
I prefer this way because it is more portable for the Client.

Reply:
Ok, I understand that the file is processed as `C` pure file without `C++` features.